src/main/java/com/gdschongik/gdsc/domain/member/domain/Member.java

```diff
@@ -345,4 +345,8 @@ public boolean isAssociate() {
     public boolean isRegular() {
         return role.equals(REGULAR);
     }
+
+    public boolean isAdmin() {
+        return role.equals(ADMIN);
```

Comment:
```suggestion
        return manageRole.equals(ADMIN);
```
ADMIN은 role이 아닌 manageRole에 해당합니다